contributed_definitions/NXcxi_ptycho.nxdl.xml

```diff
@@ -62,12 +62,10 @@
           Application definition for a ptychography experiment. This is compatible with CXI from version 1.6 (discussed with Filippe Maia) if this application definition
           is put at the top "entry" level. Above this a "cxi_version" field should be defined. The CXI format is name based, rather than class based, and so it is important
           to pay attention to the naming convention to be CXI compatible. There are duplications due to the format merger. These should be achieved by SoftLinks, 
```

Comment:
NeXus uses [HDF5 Hard Links and the `@target` attribute](http://download.nexusformat.org/doc/html/design.html#index-14) to choose which HDF5 object reference is to be considered the original.  If this *NXcxi_ptycho* definition will set a precedent for using HDF5 Soft Links, ratification of this application definition by the NIAC may involve some discussion.

Reply:
I changed this text. Any form of linking is fine, so the user should use the standard for nexus.